meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend

```diff
@@ -4,8 +4,16 @@ DEPENDS_remove = "optee-os"
 DEPENDS += "${@bb.utils.contains('MACHINE_FEATURES', 'optee', 'virtual/optee-os', '', d)}"
 
 
-SRC_URI_append_mx8m = " \
+# TEMP: Need to figure out why this is breaking mfgtool builds on mx8mp
```

Comment:
Maybe replace TEMP with TODO? Usually easier to identify stuff we need to work on later on.

Also fine to remove [TEMP] from the commit subject.

Reply:
Fixing.